modules/exploits/multi/http/agent_tesla_panel_rce.rb

```diff
@@ -0,0 +1,240 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Agent Tesla Panel Remote Code Execution',
+      'Description' => %q{
+        This module exploit a command injection vulnerability (authenticated and unauthenticated)
+        in the control center of the agent Tesla.
+
+        The Unauthenticated RCE is possible by mixing two vulnerabilities (SQLi + PHP Object Injection).
+        By observing other sources of this panel find on the Internet to watch the patch, I concluded
+        that the vulnerability is transfomed to an Authenticated RCE.
+      },
+      'Author' => [
+        'Ege Balcı <ege.balci@invictuseurope.com>', # discovery and independent module
+        'mekhalleh (RAMELLA Sébastien)' # add. targetting windows and authenticated RCE
+      ],
+      'References' => [
```

Comment:
Please link to https://www.exploit-db.com/exploits/47256 as an `EDB` reference and note that it's the original module. Thank you!

Reply:
Added EDB reference.